participants/julian-farinaccio/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1> Spooky dance moves!</h1>
+    <div class="grid-container">
+        <img src="../julian-farinaccio/images/spooky-definitelynot.gif">
+        <img src="../julian-farinaccio/images/spooky-ghost.gif">
+        <img src="../julian-farinaccio/images/spooky-pumpkin.gif">
+        <img src="../julian-farinaccio/images/spooky-light.gif">
```

Comment:
Woah! You might want to a more direct reference here, something like

`<img src="./images/..." />`

Reply:
Right! They are now updated